exercises/html-forms/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="index.html">
+      <label for="email">
```

Comment:
Every `for` attribute should have a matching `id` value on the input it is paired with. This is missing on your other inputs too, make sure you take care of those as well 🤓